package.json

```diff
@@ -3999,6 +3999,446 @@
+													},
+													{
+														"key": "D",
+														"name": "Clear all inline results",
+														"icon": "clear-all",
+														"type": "command",
+														"command": "language-julia.clearAllInlineResults"
+													},
+													{
+														"key": "K",
+														"name": "Clear all inline results in editor",
+														"icon": "clear-all",
+														"type": "command",
+														"command": "language-julia.clearAllInlineResultsInEditor"
+													},
+												]
+											},
+											{
+												"key": "s",
+												"name": "+REPL",
+												"icon": "repl",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "i",
+														"name": "Start REPL",
+														"icon": "repl",
+														"type": "command",
+														"command": "language-julia.startREPL"
+													},
+													{
+														"key": "D",
+														"name": "Stop REPL",
+														"icon": "stop",
+														"type": "command",
+														"command": "language-julia.stopREPL"
+													},
+													{
+														"key": "C",
+														"name": "Connect external REPL",
+														"icon": "vm-connect",
+														"type": "command",
+														"command": "language-julia.connectREPL"
+													},
+													{
+														"key": "s",
+														"name": "Execute code in REPL (ctrl+Enter)",
+														"icon": "play",
+														"type": "command",
+														"command": "language-julia.executeJuliaCodeInREPL"
+													},
+													{
+														"key": "m",
+														"name": "Execute code in REPL and move (alt+Enter)",
+														"icon": "debug-continue",
+														"type": "command",
+														"command": "language-julia.executeCodeBlockOrSelectionAndMove"
+													},
+													{
+														"key": "c",
+														"name": "Execute code cell in REPL and move (shift+Enter)",
+														"icon": "export",
+														"type": "command",
+														"command": "language-julia.executeCellAndMove"
+													},
+													{
+														"key": "b",
+														"name": "Execute block or selection in REPL",
+														"icon": "selection",
+														"type": "command",
+														"command": "language-julia.executeCodeBlockOrSelection"
+													},
+													{
+														"key": "f",
+														"name": "Execute file in REPL",
+														"icon": "file",
+														"type": "command",
+														"command": "language-julia.executeFile"
+													},
+													{
+														"key": "F",
+														"name": "Execute active file in REPL",
+														"icon": "file",
+														"type": "command",
+														"command": "language-julia.executeActiveFile"
+													},
+												]
+											},
+										]},
```

Comment:
This looks a little bit weird, have you tried running `npm run sort-bindings` to sort bindings?